code/solr/Solr.php

```diff
@@ -136,7 +137,7 @@ public static function service($core = null)
 
     public static function get_indexes()
     {
-        return FullTextSearch::get_indexes('SolrIndex');
+        return FullTextSearch::get_indexes('SilverStripe\FullTextSearch\Solr\SolrIndex');
```

Comment:
I think we're supposed to use `::class` now instead of a namespaced string. For example, assuming we have `use SilverStripe\FullTextSearch\Solr\` at the top of the class, you'll just use `SolrIndex::class`